utils/gyb_syntax_support/CommonNodes.py

```diff
@@ -51,4 +51,10 @@
              Child('RightBrace', kind='RightBraceToken', 
                    requires_leading_newline=True),
          ]),
+
+    Node('GarbageNodes', kind='SyntaxCollection', element='Syntax',
```

Comment:
OK, this is a bigger topic about how traversal should handle garbage and missing nodes. I believe there are two distinctly use cases for the SwiftSyntax tree
1. For use cases like a formatter, you want a fully source accurate representation of the syntax tree that includes all garbage but skips over missing nodes
2. If you are doing lightweight structural analysis of a Swift file, you want the parser to “fix-up” the syntax tree as much as possible using it’s heuristics, giving you a view of the tree that’s at least valid with respect to SwiftSyntax’s grammar. This is done by ignoring garbage nodes but walking missing tokens as if they were present.

I think these are both valid use case and sufficiently different that we shouldn’t make a decision which view should be the default. I implemented the two different views in https://github.com/apple/swift-syntax/pull/522.

Regarding the need for the `GarbageNodes` type: Theoretically, we could just use `SyntaxCollection` here, but IIRC the entire code generation pipeline is not properly set up to instantiate nodes of these base types. The `GarbageNodes` type also comes in handy for the fixed up view because it allows us to skip over all children in `GarbageSyntax`.